src/picket.ts

```diff
@@ -66,15 +68,18 @@ export class Picket {
   #authState?: AuthState;
   #chainCache: Record<string, ChainInfo> = {};
   #isAuthorizing = false;
+  static theme = CURRENT_THEME;
 
-  constructor(apiKey: string, { baseURL = BASE_API_URL }: PicketOptions = {}) {
+  constructor(apiKey: string, theme: string, { baseURL = BASE_API_URL }: PicketOptions = {}) {
```

Comment:
`theme` should be part of the `PicketOptions`
```suggestion
  constructor(apiKey: string, { baseURL = BASE_API_URL, theme = "auto" }: PicketOptions = {}) {
```